Classes/Controller/Backend/Order/DocumentController.php

```diff
@@ -104,7 +104,7 @@ public function downloadAction(Item $orderItem, string $pdfType)
                     ->withAddedHeader('Cache-Control', 'must-revalidate, post-check=0, pre-check=0')
                     ->withAddedHeader('Content-Description', 'File Transfer')
                     ->withAddedHeader('Content-Disposition', 'attachment; filename="' . $fileName . '"')
-                    ->withAddedHeader('Content-Length', $fileLen)
+                    ->withAddedHeader('Content-Length', (string)$fileLen)
```

Comment:
I would prefer casting $fileLen to string in line 100.

```suggestion
                    $fileLen = (string) $filesize($file);
```

Perhaps we could rename `$fileLen` to `$fileLength`. Pay attention to the usage some line later.